### DISCUSSION TITLE
[BUG] triple-backtick math syntax not working in existing repos

### DISCUSSION THREAD

**Discussion by nschloe** [community/community#20934] [Other Feature Feedback, Questions, & Ideas] [answered]
Since June 28, 2022, fenced codeblocks support math, see [here](https://github.blog/changelog/2022-06-28-fenced-block-syntax-for-mathematical-expressions/):
````markdown
```math
a + b = c
```
````
This works well in newly created repos, but not in existing ones such as this one right here:
```math
a + b = c
```

Screenshot:

![screenshot](https://user-images.githubusercontent.com/181628/179925789-4d1d81a4-4783-4891-b0a6-23b3e6dee79a.png)

**Comment by jfullstackdev** [comment]
I tried it in one of my old repos, this is the output

![image](https://user-images.githubusercontent.com/47092464/179993856-7641d98b-c0c3-4ea4-93ff-e999afd60ef4.png)

**Reply by nschloe** [reply]
Interesting. Then it's only _some_ old repos. (All of mine though.)

**Comment by nschloe** [comment, marked as answer]
This appears to have been fixed.
